test/napi/lib/threads.js

```diff
@@ -71,4 +81,15 @@ const assert = require('chai').assert;
     // Asynchronously GC to give the task queue a chance to execute
     setTimeout(() => global.gc(), 10);
   });
+
+  it('should handle panic in channel callback', function (cb) {
+    process.removeAllListeners("uncaughtException");
```

Comment:
`mocha` adds an `uncaughtException` listener. It needs to be removed to catch it in the test.